cqf-fhir-cr/src/test/java/org/opencds/cqf/fhir/cr/measure/dstu3/Dstu3MeasureDefBuilderTest.java

```diff
@@ -50,7 +54,7 @@ void defBuilderBooleanBasis_WithResourceExt() {
         measure.setExtension(Collections.singletonList(RESOURCE_BASIS_EXT));
 
         var def = BUILDER.build(measure);
-        assertFalse(def.isBooleanBasis());
+        assertFalse(def.groups().get(0).isBooleanBasis());
```

Comment:
Suggestion:  assert that groups.size() > 0 before making these kinds of assertions